src/main/java/org/opensearch/neuralsearch/search/query/HybridCollectorManager.java

```diff
@@ -44,9 +49,9 @@
 @RequiredArgsConstructor
 public abstract class HybridCollectorManager implements CollectorManager<Collector, ReduceableSearchResult> {
 
+    private static final int MIN_NUMBER_OF_ELEMENTS_IN_SCORE_DOC = 3;
     private final int numHits;
     private final HitsThresholdChecker hitsThresholdChecker;
-    private final boolean isSingleShard;
```

Comment:
not related to this change just a code cleanup, field isn't used anywhere in the code